.github/workflows/workflow-smoke-test.yaml

```diff
@@ -0,0 +1,61 @@
+name: Smoke Test
```

Comment:
copied from `.github/workflows/deploy-happy-stack.yml` and made into a reusable workflow so it can easily be invoked for staging and prod